src/rag/vector_store/pgvector.py

```
@@ -1,28 +1,322 @@
+        if not query_embedding:
+            logger.warning("Failed to embed query")
+            return []
+
+        # 필터 조건 생성
+        filter_clause = ""
+        filter_params: List[Any] = []
+
+        if filters:
+            filter_conditions = []
+            for key, value in filters.items():
+                filter_conditions.append(f"metadata->>%s = %s")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)